pkg/reconciler/broker/resources/dispatcher.go

```diff
@@ -176,6 +177,14 @@ func MakeDispatcherDeployment(args *DispatcherArgs) *appsv1.Deployment {
 							Name:          "http-metrics",
 							ContainerPort: 9090,
 						}},
+						SecurityContext: &corev1.SecurityContext{
+							AllowPrivilegeEscalation: ptr.Bool(false),
+							ReadOnlyRootFilesystem:   ptr.Bool(true),
+							RunAsNonRoot:             ptr.Bool(true),
+							Capabilities: &corev1.Capabilities{
+								Drop: []corev1.Capability{"all"},
```

Comment:
Hi, @mattmoor 

on eventing "core" (and serving), we have replaced `all` with `ALL`, since this is the only acceptable string check, see below:
* https://github.com/kubernetes/kubernetes/blob/6dbec8e25592d47fc8a8269c86d4b5fa838d320b/staging/src/k8s.io/pod-security-admission/policy/check_capabilities_restricted.go#L30 
* https://github.com/kubernetes/kubernetes/blob/6dbec8e25592d47fc8a8269c86d4b5fa838d320b/staging/src/k8s.io/pod-security-admission/policy/check_capabilities_restricted.go#L94

Reply:
TIL 🤔

Reply:
https://github.com/knative-sandbox/eventing-rabbitmq/pull/966